src/pybind/mgr/volumes/module.py

```diff
@@ -339,3 +454,126 @@ def _cmd_fs_volume_ls(self, inbuf, cmd):
             })
 
         return 0, json.dumps(result, indent=2), ""
+
+    def _cmd_fs_subvolume_group_ls(self, inbuf, cmd):
```

Comment:
What about access control to volumes/subvolume ls and get? I would think we'd need some kind of cephx access control for that. @liewegas any suggestions/thoughts?

Reply:
@ajarr followup with Sage please on this.

Reply:
I'll follow up on this with Sage. Meanwhile, since this is not really required for CSI driver. I'm going to defer the list commands to a subsequent PR and not delay this PR.